configs/components/pe-bolt-services.rb

```diff
@@ -31,13 +31,13 @@
       pkg.add_source("file://resources/redhat/pe-bolt-server.init", sum: "2ad1e120fa79f0d8bef8b3420dcfa229")
       pkg.add_source("file://resources/redhat/pe-bolt-server.sysconfig", sum: "273ddf6ee45968f2f96a0a7adc3b4a59")
       pkg.add_source("file://resources/redhat/pe-bolt-server.logrotate", sum: "47740a40b4c22b7d6129c51f03d14c96")
-      pkg.install_service "../pe-bolt-server.init", "../pe-bolt-server.sysconfig"
+      pkg.install_service "../pe-bolt-server.init", nil, "../pe-bolt-server.sysconfig"
```

Comment:
We don't actually want this `nil` here. The arguments are: init file, default file, service name. So this should be `pkg.install_service "../pe-bolt-server.init", "../pe-bolt-server.sysconfig", "pe-bolt-server"`.

This is currently causing the init script to be installed to `/etc/rc.d/init.d/../pe-bolt-server.sysconfig` and the sysconfig file to not be installed at all.